ArmoniK.Extension.CSharp.Client/Common/Properties.cs

```diff
@@ -272,11 +264,6 @@ public string ConnectionString
   /// </summary>
   public int ConnectionPort { get; init; } = 5001;
 
-  /// <summary>
-  ///   The TaskOptions to pass to the session or the submission session
-  /// </summary>
-  public TaskConfiguration TaskOptions { get; init; }
```

Comment:
Reasoning behind the removal ?

Reply:
I wanted to have a more consistent class and use. I juste saw, but I think we also should take out partitionIds from it, because these two caractheristcs are related to the session. The class properties has mostly connection related elements.